backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/database/Schema.kt

```diff
@@ -51,6 +53,12 @@ object Cards : IntIdTable() {
                 ) or
                 ((activationSecretHash neq null) and (codeType eq CodeType.DYNAMIC))
         }
+        check("issuerid_or_entitlementsid_not_null") {
```

Comment:
:wrench: Above check uses camel case, this uses snake case. I'd prefer the naming to be consistent.

Reply:
I generally agree, but the above check uses camel case with the first letter capitalized. 
In other tables we also have camel case with the first letter not capitalized.
And in some tables we also have snake case.

I personally like the snake case the most, because it matches the naming in the database (all the letters are in lower case there).

I think it would be nice to make the same style everywhere, but that would require applying additional migrations, so I'm not sure if it's worth it, to be honest.